2006Redone Server/src/redone/net/packets/impl/ClickItem.java

```diff
@@ -143,16 +144,19 @@ public void processPacket(Client player, int packetType, int packetSize) {
 			break;
 			
 		case 2677:
+			if (Constants.CLUES_ENABLED)
 			player.getItemAssistant().deleteItem(itemId, 1);
 			TreasureTrails.addClueReward(player, 0);
 			break;
 
 		case 2678:
+			if (Constants.CLUES_ENABLED)
 			player.getItemAssistant().deleteItem(itemId, 1);
 			TreasureTrails.addClueReward(player, 1);
 			break;

```

Comment:
This is just going to prevent the clues from being deleted but will still give the rewards.

_same for the other 2 usages_

Should be:
```java

			if (Constants.CLUES_ENABLED) {
				player.getItemAssistant().deleteItem(itemId, 1);
				TreasureTrails.addClueReward(player, 1);
			}
			break;
```